_includes/payment-order-other-features.md

```diff
@@ -596,17 +596,48 @@ Content-Type: application/json
+
+| {% icon check %} | └─➔&nbsp;`payeeId`             | `string`      | This is the unique id that identifies this payee (like merchant) set by Swedbank Pay.                                                                                                                                                                                                                                                                                           |
+| {% icon check %} | └➔&nbsp;`payeeReference`       | `string(30)` | A unique reference from the merchant system. It is set per operation to ensure an exactly-once delivery of a transactional operation. See [`payeeReference`][payee-reference] for details. In Invoice Payments `payeereference` is used as an invoice/receipt number.                 
+|
+| {% icon check %} | └─➔&nbsp;`payeeName`           | `string`      | The payee name (like merchant name) that will be displayed to consumer when redirected to Swedbank Pay.                                                                                                                                                                                                                                                                                                              |
+| {% icon check %} | └─➔&nbsp;`productCategory`     | `string`      | A product category or number sent in from the payee/merchant. This is not validated by Swedbank Pay, but will be passed through the payment process and may be used in the settlement process.                                                                                                                                                                                                                           |
+| {% icon check %} | └─➔&nbsp;`orderReference`      | `String(50)`  | The order reference should reflect the order reference found in the merchant's systems.                                                                                                                                                                                                                                                                                                                                  |
+| {% icon check %} | └─➔&nbsp;`subsite`             | `String(40)`  | The subsite field can be used to perform [split settlement][split-settlement] on the payment. The subsites must be resolved with Swedbank Pay [reconciliation][settlement-and-reconciliation] before being used.                                                                                                                                                                                                                                                                                                                             |
```

Comment:
Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5741678)